geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionImpl.java

```diff
@@ -325,9 +378,10 @@ public InternalDistributedMember getLocalMember() {
       if (sentBytes == 0) {
         membership.checkCancelled();
```

Comment:
looks like this could just call the local `checkCancelled`

Reply:
oh - good catch!  I'll do that